### ISSUE TITLE
Create an OCR app that can read braille symbols and convert it to vibration patterns.

### ISSUE THREAD

**Issue by sunn-e** [aamba/Aamba#20]
I got this inspiration after reading this article.
https://www.cnet.com/news/vibrating-touch-screen-enables-braille-reading/
It would be great if we could leverage machine learning/deep learning or other AI algorithms to detect the symbols and then convert it to vibration patterns.

**Comment by Roymprog** [comment]
This is not a "good first issue" IMO

**Comment by hgayan7** [comment]
Similar to this https://www.researchgate.net/publication/254051825_Reading_Braille_on_mobile_phones_A_fast_method_with_low_battery_power_consumption

**Comment by sunn-e** [comment]
Yes, This will be an awesome project to implement. I will look if the authors are willing to collaborate or share their resources for this project. Thanks for the link @hgayan7 , you are awesome.

**Comment by hgayan7** [comment]
Most welcome @sunn-e
